modules/gdscript/editor/gdscript_highlighter.h

```diff
@@ -49,6 +49,7 @@ class GDScriptSyntaxHighlighter : public EditorSyntaxHighlighter {
 
 	HashMap<StringName, Color> keywords;
 	HashMap<StringName, Color> member_keywords;
+	HashMap<StringName, Color> global_functions;
```

Comment:
If the colour is always to same can we not use a set instead, or is there a situation where it would need to differ?

Reply:
That's true, they may be able to be fetched at the start instead of being cleared and refreshed every `_refresh_cache()`.

Reply:
I found the implementation weird, but thought that there might be a good reason behind it. Isn't member_keywords always the same color, too?

Reply:
I looked at the code again and... huh. I thought the current Script changed something (and it can be argued it probably should), but it looks like there's room for optimisation for both, I theorise. Maybe for some other time?

Reply:
Used a HashSet, seemed like the appropriate data structure.